### ISSUE TITLE
Research Documents In Navigation View - Wish

### ISSUE THREAD

**Issue by siliconserf** [olivierkes/manuskript#276]
Working on a story idea, I found that I need a feature in _Navigation_ I will call _Research_. Research files are not typically closely related to any particular plot, so they don't belong in _Plots_. Nor do they fit into _Outline_. In some cases they might amount to the literary equivalent of "doodling". 

This feature would work much like _Outlines_, but other than titles, date created, and a compile option, no other meta-data is needed. The ability to index image files, (or allow them to be embedded in documents) is desirable. No other section, such as _Edit_ would need to reflect the documents in _Research_, as it does with _Outlines_.

**Comment by olivierkes** [comment]
Thanks for the suggestion. This is connected to #22.

The research folder is still in the roadmap, but you add a new idea: to make it a separate item in Navigation. It makes sense. I don't know when this will happen: I still have things to clean under the hood to make it a reality.

**Comment by Exitr** [comment]
Good idea. Research integration is, of course, one of the things Scrivener is well known for; having a comparable feature in Manuskript would be a nice addition.

**Comment by Hard-Wired** [comment]
I was wondering about this omission. It would be a grateful addition for sure. It is currently the primary reason haven't switched over yet. I do a huge amount of research before I really start writing and it would be helpful to be able to reference all my notes, lists, images, and links. In fact I'd like to be able to link directly (via markup?) to research information within the project from any other section so I can get back to it quickly while I am writing and fact checking. External links are not as useful.

**Comment by EugeneUvin** [comment]
Hi, I also think this is obviously missing - I've watched workflows in the other (very famous) app, and they all add research separately. Looks like having a separate tab on the left would be what is needed.
